script/openchange_user_cleanup.py

```diff
@@ -63,20 +64,60 @@ def active_openchange_users(self):
 
 
 class ImapCleaner(object):
-    def __init__(self, imap_host="127.0.0.1", imap_port=143, dry_run=False,
-                 samba_helper=SambaOCHelper()):
-        self.imap_host = imap_host
-        self.imap_port = imap_port
+    def __init__(self, dry_run=False, samba_helper=SambaOCHelper()):
         self.dry_run = dry_run
         self.samba_helper = samba_helper
+        self.system_defaults_file = "/etc/sogo/sogo.conf"
+        self.user_defaults_file = \
+            os.path.expanduser("~sogo/GNUstep/Defaults/.GNUstepDefaults")
```

Comment:
You expanduser ~sogo which I presume is Zentyal specifc. Can you remove this hardcoded path?